src/components/JazzIcon/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from 'react';
+import md5 from 'md5';
+// @ts-ignore
+import jazzicon from '@metamask/jazzicon';
+
+type JazziconProps = {
+  address: string;
+  diameter: number;
+};
```

Comment:
I'll be doing `interface JazzIconProps extends HTMLAttributes<HTMLDivElement>`, I readed about `React.FC<JazzIconProps>` but it was considered an anti-pattern because it allowed components to accept children even when they weren't supposed to.

Seems that the FC hate has disappeared starting from React 18 and TS 5.1 as it children is no longer included in the props type, I think we can discuss this on tomorrows daily :)